### PR TITLE
Release/5.6.0

### PR DESCRIPTION
**Pull request by taquitos**
### Other Changes
* CleverTapID integration (#593) via Joshua Liebowitz (@taquitos)
* Rename package name of purchase tester app to com.revenuecat.purchases_sample (#592) via Toni Rico (@tonidero)
* Fix prepare next version (#591) via Toni Rico (@tonidero)

### CODE REVIEW

File: CHANGELOG.latest.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-### API Updates
-* `Transaction` class properties `revenuecatId` and `productId` have been deprecated. Use `transactionIdentifier` and `productIdentifier` respectively instead (#587)
-
 ### Other Changes
-* Remove billing client 3 support (#567) via beylmk (@beylmk)
+* CleverTapID integration (#593) via Joshua Liebowitz (@taquitos)
+* Rename package name of purchase tester app to com.revenuecat.purchases_sample (#592) via Toni Rico (@tonidero)
+* Fix prepare next version (#591) via Toni Rico (@tonidero)
```
Comment (tonidero) [inline CHANGELOG.latest.md:4]: I would probably remove this one from the changelog? I don't consider it user-facing

**Review by tonidero — APPROVED** [review]

File: CHANGELOG.latest.md (hunk 1 above)
Comment (taquitos) [reply CHANGELOG.latest.md:4]: 👍

File: CHANGELOG.latest.md [hunk 2]
```diff
@@ -1,5 +1,3 @@
-### API Updates
-* `Transaction` class properties `revenuecatId` and `productId` have been deprecated. Use `transactionIdentifier` and `productIdentifier` respectively instead (#587)
-
 ### Other Changes
-* Remove billing client 3 support (#567) via beylmk (@beylmk)
+* CleverTapID integration (#593) via Joshua Liebowitz (@taquitos)
```
[the remark this answers is not in the document]
Comment (taquitos) [reply CHANGELOG.latest.md:2]: Oooo I think I used the wrong tag, then.

**Comment by vegaro** [comment]
can we merge this? I think the release was finished manually right?

**Comment by tonidero** [comment]
Yeah I think we should merge it. Will hold until @taquitos takes a look just in case

**Pull request merged**